lib/Doctrine/DBAL/Schema/Index.php

```diff
@@ -196,7 +196,7 @@ public function spansColumns(array $columnNames)
      *
      * @return bool
      */
-    public function isFullfilledBy(Index $other)
+    public function isFulfilledBy(Index $other)
```

Comment:
That's a BC-break. Please create a mispelt method that calls this one, and throws a deprecation notice before doing so.

Reply:
Thank you! Will do.